examples/app-mvvm/app/Application.js

```diff
@@ -0,0 +1,23 @@
+/**
+ * The main application class. An instance of this class is created by app.js when it calls
+ * Ext.application(). This is the ideal place to handle application launch and initialization
+ * details.
+ */
+Ext.define('CF.Application', {
+    extend: 'Ext.app.Application',
+    
+    name: 'CF',
+    
+    requires: [
+        'CF.view.summit.Grid',
+        'CF.view.summit.Chart'
+    ],
+
+    stores: [
+        // TODO: add global / shared stores here
+    ],
+    
+    launch: function () {
```

Comment:
Right. Or should we leave the ``launch`` function with a more speaking comment, that this is the point to add your custom code when launching the app?
I am fine with both - removing or commenting.